### ISSUE TITLE
Does bambi support (1| A/B) grouping syntax?

### ISSUE THREAD

**Issue by aaelony** [bambinos/bambi#566]
The `brms` package allows a syntax of `y ~ 1 + (1 | re_1 / re_2 / re_3)`. Is this supported in `bambi`?

```
(1 | A/B) translates to (1 | A) + (1 | A:B) where A:B simply means creating a new grouping factor with the levels of A and B pasted together.
```
Thank-you

references:
 - https://discourse.mc-stan.org/t/brms-approach-to-stan-models-for-nested-versus-non-nested-multi-level-models/9278

**Comment by zwelitunyiswa** [comment]
I have used the expanded nested syntax and it has worked fine for me.

**Comment by aaelony** [comment]
I understand.  Is the expanded syntax prefered over the concise brms version?

**Comment by zwelitunyiswa** [comment]
I would recommend it. The concise syntax can have an issue with ordering if
you have multiple nested terms, and not behave as you would expect.
Debugging this can be a pain. The expanded notation is explicit and easier
to debug IMHO.

**Comment by aaelony** [comment]
Thanks for clarifying this.  I think I would prefer concision and good debugging ;).  Maybe that can be a future feature?

For now, I'll take the expanded syntax approach and create new variables that are the pastes of the A and B levels above.

**Comment by tomicapretto** [comment]
@aaelony yes, this syntax is supported. Do you have an example where you try to use it? Is it working as expected?

As @zwelitunyiswa says, there are times when expanded syntax is preferred.

**Comment by tomicapretto** [comment]
I'm closing the issue since I consider the question to be answered. Thanks!
